modules/src/main/java/eu/itesla_project/modules/constraints/ConstraintsModifier.java

```diff
@@ -177,21 +172,20 @@ else if ( violatedBranch instanceof TwoWindingsTransformer )
                     } else {
                         LOGGER.warn("State {}: cannot change current limit of branch {}: no branch with this id in the network", 
                                 initialStateId, 
-                                violatedBranch.getId());
+                                violation.getSubjectId());
                     }
```

Comment:
Here the code works on a different network state, that could not have the branch